src/lib/account/solana.ts

```diff
@@ -0,0 +1,6 @@
+import type { DeriveAccountIdsAndMetadata, DeriveAccountIdsAndMetadataReturn } from './account'
+
+export const deriveSolanaAccountIdsAndMetadata: DeriveAccountIdsAndMetadata = () => {
+  // @TODO: support Solana account derivation
+  return {} as DeriveAccountIdsAndMetadataReturn
+}
```

Comment:
Same, TS is not happy if we donc add it to `deriveAccountIdsAndMetadataForChainNamespace` because `ChainNamespaceKey` does include the solana namespace